book/src/SUMMARY.md

```diff
@@ -5,6 +5,10 @@
   * [Example Client: Web Wallet](getting-started/webwallet.md)
 * [Programming Model](programs/README.md)
   * [Example: Tic-Tac-Toe](programs/tictactoe.md)
+  * [Transaction](api-reference/transaction-api.md)
+  * [Instruction](api-reference/instruction-api.md)
+  * [JSON RPC API](api-reference/jsonrpc-api.md)
+  * [JavaScript API](api-reference/javascript-api.md)
```

Comment:
Perhaps first we should distinguish application development from on-chain program development.